shared/domains/gi/db.js

```diff
@@ -37,8 +39,14 @@ export default (sbp('sbp/selectors/register', {
   'gi.db/set': function (key: string, value: string): Promise<void> {
     return Promise.resolve(sbp('okTurtles.data/set', key, value))
   },
+  'gi.db/delete': function (key: string): Promise<void> {
+    return Promise.resolve(sbp('okTurtles.data/delete', key))
+  },
   'gi.db/log/logHEAD': function (contractID: string): string {
-    return `${contractID}-HEAD`
+    return `${contractID}${headSuffix}`
+  },
+  'gi.db/log/contractIdFromLogHEAD': function (key: string): string | boolean {
+    return key.endsWith(headSuffix) ? key.slice(0, -headSuffix.length) : false
```

Comment:
Wouldn't it make more sense to have the return type be `?string` and return `null` / `undefined` instead of `false`?